hack/migrate-to-antrea.sh

```diff
@@ -0,0 +1,256 @@
+    echo "kubectl rollout status ds/antrea-agent -n kube-system --timeout=300s --kubeconfig $KUBECONFIG"
+else
+    kubectl rollout status ds/antrea-agent -n kube-system --timeout=300s --kubeconfig "$KUBECONFIG"
+    echo "Antrea is ready"
+fi
+if [ $NETWORK_POLICY == true ]; then
+    echo "========== Converting NetworkPolicy to Antrea NetworkPolicy =========="
+    if [ $DRY_RUN == true ]; then
+        echo "./bin/antctl migrate convert-networkpolicy --from $FROM --kubeconfig $KUBECONFIG"
+    else
+        ./bin/antctl migrate convert-networkpolicy --from "$FROM" --kubeconfig "$KUBECONFIG"
+    fi
+fi
+echo "========== Restarting the sandboxes =========="
+if [ $DRY_RUN == true ]; then
+    echo "kubectl apply -f ./build/yamls/antrea-migrator.yml --kubeconfig $KUBECONFIG"
+else
+    trap "kubectl delete -f ./build/yamls/antrea-migrator.yml --kubeconfig $KUBECONFIG" EXIT
+    kubectl apply -f ./build/yamls/antrea-migrator.yml --kubeconfig "$KUBECONFIG"
+fi
+if [ $DRY_RUN == true ]; then
+    echo "kubectl rollout status ds/antrea-migrator -n kube-system --timeout=300s --kubeconfig $KUBECONFIG"
+else
+    echo "Waiting for sandboxes restart to be finished"
+    kubectl rollout status ds/antrea-migrator -n kube-system --timeout=300s --kubeconfig "$KUBECONFIG"
+fi
+
+echo "========== The restart process is finished, begin to clean the old CNI =========="
+
+if [ $FROM == "calico" ]; then
+    if [ $DRY_RUN == true ]; then
+        echo "kubectl delete -f $CALICO_MANIFEST --kubeconfig $KUBECONFIG"
+    else
+        echo "Cleaning Calico resources"
+        kubectl delete -f $CALICO_MANIFEST --kubeconfig "$KUBECONFIG"
+    fi
+elif [ $FROM == "flannel" ]; then
+    if [ $DRY_RUN == true ]; then
+        echo "kubectl delete -f $FLANNEL_MANIFEST"
+    else
+        echo "Cleaning Flannel resources"
+        kubectl delete -f $FLANNEL_MANIFEST
+    fi
+fi
+
+echo "========== Cleaning iptabes rules =========="
+antrea_agents=$(kubectl get pods -n kube-system -l component=antrea-agent -o jsonpath='{.items[*].metadata.name}' --kubeconfig "$KUBECONFIG")
+if [ $DRY_RUN == true ]; then
+    if [ $FROM == "calico" ]; then
+        echo "kubectl exec -n kube-system {ANTREA_AGENT} -- /bin/bash -c 'iptables-save | grep -v cali | iptables-restore' --kubeconfig $KUBECONFIG"
+    else
+        echo "kubectl exec -n kube-system {ANTREA_AGENT} -- /bin/bash -c '
+                    ip link delete cni0 || true
+                    ip iink delete flannel.1 || true
+                    iptables-save | grep -v flannel | iptables-restore || true' --kubeconfig $KUBECONFIG || true"
+    fi
+else
+    if [ $FROM == "calico" ]; then
+        for agent in $antrea_agents; do
+            kubectl exec -n kube-system "$agent" -- /bin/bash -c 'iptables-save | grep -v cali | iptables-restore' --kubeconfig "$KUBECONFIG"
+        done
+    else
+        for agent in $antrea_agents; do
+            kubectl exec -n kube-system "$agent" -- /bin/bash -c '
```

Comment:
Better to keep the format the same as the above commands in dryrun mode.
```
 kubectl exec -n kube-system "${ANTREA_AGENT} "
```
Change one of them to keep the format consistent.